modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -842,6 +862,83 @@ private HttpServletRequest _getDDMStructureHttpServletRequest(
 		return dynamicServletRequest;
 	}
 
+	private Date _getExpirationDate(
+			UploadPortletRequest uploadPortletRequest, User user)
```

Comment:
You're passing the user so that you can get its time zone. Why not pass the time zone instead?